### ISSUE TITLE
Chunking of files for storage systems with file size restriction.

### ISSUE THREAD

**Issue by Cr0nixx** [ncw/rclone#1154]
This is a proposal to add transparent chunking for files to the crypt storage system.

This would for example allow someone to upload files bigger than 50GB to ACD. This would be a configuration option in the crypt config. Now how the file information would be saved is up for ideas. Have the information embedded into the chunks or other. 

Example:

Uploading a 100 GB file in a crypt remote with chunk size of 1GB would result in 100 files on the share with rclone showing one physical file.

Would love to see this implemented.
This would make rclone an excellent replacement for acd_cli with cryfs, gocrypt or similar.

**Comment by CrossR** [comment]
Think this is a duplicate of ticket #497.

That said, I'll post here as well to say I'd love this.
I'm uploading some OS images to ACD, and between small internet drops and Amazon errors, I've uploaded about 200GB for only 80GB of files, and I've still got 40GB left. It'll crash randomly when uploading and then restart.

I assume that if it were chunked, it would also have the advantages that if I upload chunks 1-50 and then lost internet, I can carry on at chunk 51 etc, which would help uploading files even under 50GB.

If chunks were in parallel, it would also help the weird speed issues ACD has where uploading a single file is slower than multiple ones some of the time.

**Comment by Cr0nixx** [comment]
Oh, I guess yeah. I'll close this then.

**Comment by ncw** [comment]
Can you add your support to issue #497 please?  The more subscribers I see to an issue, the more popular I know it is!

Thanks
